tasks/src/text-to-image/data.ts

```diff
@@ -0,0 +1,86 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			// TODO write proper description
+			description: "Caltech-UCSD Birds-200-2011 is an extended version of the CUB-200 dataset, with roughly double the number of images per class and new part location annotations.",
```

Comment:
Can you add datasets that are on Hub and are rather canonical? Couple of examples you could pick from: 
https://huggingface.co/datasets/red_caps
https://huggingface.co/datasets/conceptual_captions
https://huggingface.co/datasets/conceptual_12m
they're easier to load and are generic 🙂

Reply:
We usually try to have 2-3 models so I'd be in favor of removing 🙂